lib/pages/dashboard/dashoard_page.dart

```diff
@@ -61,20 +63,30 @@ class _DashboardPageState extends State<DashboardPage> {
       create: (context) => _favoriteBloc,
       child: BlocBuilder<FavoriteBloc, FavoriteState>(
         builder: (context, state) {
-          return (state is FavoriteLoadedState)
-              ? ListView.builder(
-                  shrinkWrap: true,
-                  itemCount: 6,
-                  itemBuilder: (context, index) {
-                    return _buildDashboardItem(
-                      index,
-                      state,
-                    );
-                  },
-                )
-              : SizedBox();
-        },
-      ),
+          return CustomScrollView(
+            slivers: <Widget>[
+              SliverAppBar(
+                pinned: false,
+                snap: false,
+                floating: true,
+                title: Text('Tableau de bord'),
```

Comment:
As the dashboard is the main page, I think that reusing the application's name is a good pattern. In other pages we will use the a real title